CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated 
 ### Removed
 ### Fixed
+    - Issue 144 - The timeseries Lambda function logs the user_ip twice
```

Comment:
let's actually remove the changelog entry here because it's fixing a bug for #126

Reply:
Done!